accessibility-checker-extension/src/ts/devtools/ReportRow.tsx

```diff
@@ -70,6 +76,28 @@ export default class ReportRow extends React.Component<IReportRowProps, IReportR
         }
     }
 
+    learnMoreClickHandler = (e:any, item:IReportItem)=>{
+        e.preventDefault();
+        e.stopPropagation();
+        this.props.getItem(item);
+    }
+
+    learnMoreKeyDownHandler = (e:any, item:IReportItem)=>{
+        if (e.keyCode === 13) {
+            e.preventDefault();
+            e.stopPropagation();
+            this.props.getItem(item);
+        }
+    }
+
+    learnMoreRef = (item: IReportItem) => {
```

Comment:
Same comment as for `learnMoreClickHandler`

Reply:
changed